src/Symfony/Bridge/Doctrine/Tests/Form/DoctrineOrmTypeGuesserTest.php

```diff
@@ -11,16 +11,40 @@
 
 namespace Symfony\Bridge\Doctrine\Tests\Form;
 
+use Doctrine\DBAL\Types\Types;
 use Doctrine\ORM\Mapping\ClassMetadata;
 use Doctrine\Persistence\ManagerRegistry;
 use Doctrine\Persistence\ObjectManager;
 use PHPUnit\Framework\TestCase;
 use Symfony\Bridge\Doctrine\Form\DoctrineOrmTypeGuesser;
 use Symfony\Component\Form\Guess\Guess;
+use Symfony\Component\Form\Guess\TypeGuess;
 use Symfony\Component\Form\Guess\ValueGuess;
 
 class DoctrineOrmTypeGuesserTest extends TestCase
 {
+    /**
+     * @dataProvider requiredType
```

Comment:
no need for a provider here since there is only one item in the provided list

Reply:
Do you want me to add more items to this list?

Reply:
only if it makes sense to improve the code coverage
otherwise, no need for the provider

Reply:
I've added one more test to also cover DateTimeImmutable :-)